### ISSUE TITLE
Crash in RPCAPI

### ISSUE THREAD

**Issue by mattias-p** [zonemaster/zonemaster-backend#559]
When I run the post-installation sanity-check for Backend it simply says `Internal Server Error`. And in /var/log/zonemaster/zm-starman-error.log there's this message:
```
Can't locate object method "compile" via package "JSON::Validator" at /usr/local/share/perl/5.24.1/Zonemaster/Backend/RPCAPI.pm line 530.
```
This is on Debian 9 (stretch).

**Comment by sandoche2k** [comment]
I get the same on CentOS

**Comment by mattias-p** [comment]
It's an API change in JSON::Validator. I'll make a PR shortly.

**Comment by matsduf** [comment]
Will be resolved by #562.
